cmd/containeragent/initialize/command.go

```diff
@@ -138,12 +152,55 @@ func (c *initCommand) Run(ctx *cmd.Context) error {
 			return nil
 		})
 	}
+
+	if err := c.ContainerAgentPebbleConfig(); err != nil {
+		return err
+	}
+
 	doCopy("/opt/pebble", path.Join(c.binDir, "pebble"))
 	doCopy("/opt/containeragent", path.Join(c.binDir, "containeragent"))
 	doCopy("/opt/jujuc", path.Join(c.binDir, "jujuc"))
 	return eg.Wait()
 }
 
+// ContainerAgentPebbleConfig is responsible for generating the container agent
+// pebble service configuration.
+func (c *initCommand) ContainerAgentPebbleConfig() error {
+	pebbleConfigTemplate := `
+---
+summary: Juju container agent service
+services:
+  container-agent:
+    override: replace
+    summary: Juju container agent
+    command: %s unit --data-dir %s --append-env "PATH=$PATH:%s" --show-log %s
+    startup: enabled
```

Comment:
you could think about using a struct here and yaml.Marshal'ing it. that's what I'm doing in #14640. might help prevent future syntax errors but it's not strictly necessary

Reply:
Would be better if Pebble provided us the struct so we can stay in sync.

Reply:
Working on this now